2020_10_02-traefik-load-balancing/ansible/roles/kubernetes/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 - include_tasks: install.yml
+- include_tasks: install_ansible.yml
```

Comment:
should be install_ansible_requirements.yml or something like that but not install_ansible.yml because then we assume it installs ansible, which is not the case.